core/geometry/src/geometry3d/Ray3d.ts

```diff
@@ -254,8 +293,10 @@ export class Ray3d implements BeJSONFunctions {
    * Convert an Angle to a JSON object.
    * @return {*} [origin,normal]
    */
-  public toJSON(): any { return { origin: this.origin.toJSON(), direction: this.direction.toJSON() }; }
-  /** Create a new ray from json object.  See `setFromJSON` for json structure; */
+  public toJSON(): any {
```

Comment:
Please change the first sentence of the doc for `toJSON` -> "Construct a JSON object from this Ray3d."